vpr/src/draw/draw_mux.cpp

```diff
@@ -0,0 +1,133 @@
+/*draw_mux.cpp contains all functions that draw muxes.*/
```

Comment:
Good to put a space after /* and before */; easier to read and easier to search for.